src/components/Form/Form.tsx

```diff
@@ -0,0 +1,117 @@
+import React, { useState } from 'react';
+import { Todo } from '../../types/Todos';
+import { DefaultErrorsState, User } from '../../types';
+import { TextField } from '../TextField/TextField';
+import { getRandomDigits } from '../../utils';
+import { SelectUser } from '../SelectUser';
+import todos from '../../api/todos';
```

Comment:
The `todos` are imported directly from the server. This violates the requirement to prepare data in the App component and pass it down to child components. Consider removing this import and passing necessary data as props from the App component.